k8s/helm_charts2/values.yaml

```diff
@@ -273,9 +273,6 @@ filer:
   # Limit sub dir listing size (default 100000)
   dirListLimit: 100000
 
-  # Turn off directory listing
-  disableDirListing: false
```

Comment:
This is an unrelated change.

Reply:
@chrislusf  This is a duplicate of IDE highlighted